### PR TITLE
Readme documentation for LF10

### PR DESCRIPTION
**Pull request by jeremie-lucca**

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1748/storybook

**Comment by CCNET-iLucca** [comment]
[Tests d'interfaces](https://recette-auto.lucca.fr/lucca-front-163-5e7c0f1c-54cc-4de7-a7dd-07608cc25a6a/ui-diff.html)

File: README.md [hunk 1]
```diff
@@ -23,142 +20,103 @@ npm install @lucca-front/scss --save
...
+@forward '@lucca-front/scss/src/components/tableOfContent';
+@forward '@lucca-front/scss/src/components/tableSorted';
+@forward '@lucca-front/scss/src/components/tableSticked';
+@forward '@lucca-front/scss/src/components/tag';
+@forward '@lucca-front/scss/src/components/textfield';
+@forward '@lucca-front/scss/src/components/timeline';
+@forward '@lucca-front/scss/src/components/title';
+@forward '@lucca-front/scss/src/components/titleSection';
+@forward '@lucca-front/scss/src/components/toast';
+@forward '@lucca-front/scss/src/components/util';
```
Comment (vvalentin-lucca) [inline README.md:84]: Won't it be hard to maintain this list here?

**Review by GuillaumeNury — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jeremie-lucca) [reply README.md:84]: Agreed.
Let's merge it for the release need and if we fail to maintain it, we'll try to improve it.

**Pull request merged**